src/main/java/wolox/training/repository/BookRepository.java

```diff
@@ -0,0 +1,12 @@
+package wolox.training.repository;
```

Comment:
I think that this should not be in this PR, since this PR corresponds to the controller and not to the repository